cmd/zdb/Makefile.am

```diff
@@ -12,3 +13,5 @@ zdb_LDADD = \
 	libzpool.la \
 	libzfs_core.la \
 	libnvpair.la
+
+zdb_LDADD += $(LIBCRYPTO_LIBS)
```

Comment:
adding openssl for key folding etc, right

Reply:
Yep!